Mathlib/Probability/Kernel/Composition/ParallelComp.lean

```diff
@@ -47,70 +34,122 @@ namespace ProbabilityTheory.Kernel
 
 variable {α β γ δ : Type*} {mα : MeasurableSpace α} {mβ : MeasurableSpace β}
   {mγ : MeasurableSpace γ} {mδ : MeasurableSpace δ}
+  {κ : Kernel α β} {η : Kernel γ δ} {x : α × γ}
 
-section ParallelComp
-
+open Classical in
 /-- Parallel product of two kernels. -/
 noncomputable
 def parallelComp (κ : Kernel α β) (η : Kernel γ δ) : Kernel (α × γ) (β × δ) :=
```

Comment:
should this be an `irreducible_def` now that it's an `if` (which means we will almost never use the definition, only its API)?